app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
```

Comment:
The `command.split(" ")` method is called twice in the code. You can store the result in a variable and reuse it to make the code more efficient.